example/src/main/java/org/apache/rocketmq/example/benchmark/timer/TimerConsumer.java

```diff
@@ -151,6 +166,30 @@ private Options buildCommandlineOptions(Options options) {
         opt.setRequired(false);
         options.addOption(opt);
 
+        opt = new Option("w", "threadCount", true, "Thread count, Default: 20");
+        opt.setRequired(false);
+        options.addOption(opt);
+
+        opt = new Option("m", "msgTraceEnable", true, "Message Trace Enable, Default: false");
+        opt.setRequired(false);
+        options.addOption(opt);
+
+        opt = new Option("g", "group", true, "Consumer group name, Default: benchmark_consumer");
+        opt.setRequired(false);
+        options.addOption(opt);
+
+        opt = new Option("a", "aclEnable", true, "Acl Enable, Default: false");
+        opt.setRequired(false);
+        options.addOption(opt);
+
+        opt = new Option("ak", "accessKey", true, "Acl access key, Default: rocketmq2");
```

Comment:
Having defaults for access key/secret might not be a good idea.

Reply:
Thanks for your review, I’ll exchange their default value like existed code.